```
======================================================================
ERROR: test_basic_application_schema (corehq.apps.export.tests.test_export_data_schema.TestBuildingCaseSchemaFromApplication)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/apps/export/tests/test_export_data_schema.py", line 463, in test_basic_application_schema
    schema = CaseExportDataSchema.generate_schema_from_builds(self.domain, self.case_type)
  File "/mnt/corehq/apps/export/models/new.py", line 1049, in generate_schema_from_builds
    include_parent_properties=False
  File "/mnt/corehq/apps/app_manager/util.py", line 287, in get_case_properties
    per_type_defaults = get_per_type_defaults(app.domain, case_types)
  File "/mnt/corehq/apps/app_manager/util.py", line 305, in get_per_type_defaults
    callcenter_case_type = get_call_center_case_type_if_enabled(domain)
  File "/mnt/corehq/util/quickcache.py", line 312, in inner
    return helper(*args, **kwargs)
  File "/mnt/corehq/util/quickcache.py", line 100, in __call__
    content = self.fn(*args, **kwargs)
  File "/mnt/corehq/apps/callcenter/utils.py", line 281, in get_call_center_case_type_if_enabled
    config = domain_object.call_center_config
AttributeError: 'NoneType' object has no attribute 'call_center_config'
```
and

```
======================================================================
ERROR: test_simple_case_management (corehq.apps.userreports.tests.test_app_manager_integration.AppManagerDataSourceConfigTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1224, in patched
    return func(*args, **keywargs)
  File "/mnt/corehq/apps/userreports/tests/test_app_manager_integration.py", line 33, in test_simple_case_management
    data_sources = get_case_data_sources(app)
  File "/mnt/corehq/apps/userreports/app_manager.py", line 22, in get_case_data_sources
    return {case_type: get_case_data_source(app, case_type) for case_type in app.get_case_types() if case_type}
  File "/mnt/corehq/apps/userreports/app_manager.py", line 22, in <dictcomp>
    return {case_type: get_case_data_source(app, case_type) for case_type in app.get_case_types() if case_type}
  File "/mnt/corehq/apps/userreports/app_manager.py", line 27, in get_case_data_source
    prop_map = get_case_properties(app, [case_type], defaults=DEFAULT_CASE_PROPERTY_DATATYPES.keys())
  File "/mnt/corehq/apps/app_manager/util.py", line 287, in get_case_properties
    per_type_defaults = get_per_type_defaults(app.domain, case_types)
  File "/mnt/corehq/apps/app_manager/util.py", line 305, in get_per_type_defaults
    callcenter_case_type = get_call_center_case_type_if_enabled(domain)
  File "/mnt/corehq/util/quickcache.py", line 312, in inner
    return helper(*args, **kwargs)
  File "/mnt/corehq/util/quickcache.py", line 100, in __call__
    content = self.fn(*args, **kwargs)
  File "/mnt/corehq/apps/callcenter/utils.py", line 280, in get_call_center_case_type_if_enabled
    domain_object = Domain.get_by_name(domain)
  File "/mnt/corehq/util/quickcache.py", line 312, in inner
    return helper(*args, **kwargs)
  File "/mnt/corehq/util/quickcache.py", line 201, in __call__
    return super(SkippableQuickCache, self).__call__(*args, **kwargs)
  File "/mnt/corehq/util/quickcache.py", line 100, in __call__
    content = self.fn(*args, **kwargs)
  File "/mnt/corehq/apps/domain/models.py", line 538, in get_by_name
    domain = _get_by_name(stale=(not strict))
  File "/mnt/corehq/apps/domain/models.py", line 530, in _get_by_name
    result = cls.view("domain/domains", key=name, reduce=False, include_docs=True, **extra_args).first()
  File "/vendor/lib/python2.7/site-packages/couchdbkit/schema/base.py", line 310, in view
    return db.view(view_name,
  File "/vendor/lib/python2.7/site-packages/mock.py", line 700, in __getattr__
    raise AttributeError("Mock object has no attribute %r" % name)
AttributeError: Mock object has no attribute 'view'
```

--- a/corehq/apps/callcenter/utils.py
+++ b/corehq/apps/callcenter/utils.py
@@ -278,6 +278,9 @@ def get_call_center_cases(domain_name, case_type, user=None):
 @quickcache(['domain'])
 def get_call_center_case_type_if_enabled(domain):
     domain_object = Domain.get_by_name(domain)
+    if not domain_object:
+        return
+
     config = domain_object.call_center_config
     if config.enabled and config.config_is_valid():
         return config.case_type

--- a/corehq/apps/userreports/tests/test_app_manager_integration.py
+++ b/corehq/apps/userreports/tests/test_app_manager_integration.py
@@ -1,7 +1,7 @@
 import json
 import os
 from datetime import datetime
-from mock import patch
+from mock import patch, Mock
 from django.test import SimpleTestCase
 from corehq.apps.app_manager.models import Application
 from corehq.apps.userreports.app_manager import get_case_data_sources, get_form_data_sources
@@ -24,6 +24,7 @@ class AppManagerDataSourceConfigTest(SimpleTestCase):
             return json.loads(f.read())
 
     @patch('corehq.apps.userreports.specs.datetime')
+    @patch('corehq.apps.app_manager.util.get_per_type_defaults', Mock(return_value={}))
     def test_simple_case_management(self, datetime_mock):
         fake_time_now = datetime(2015, 4, 24, 12, 30, 8, 24886)
         datetime_mock.utcnow.return_value = fake_time_now

